jenkins/check_copyright.py

```diff
@@ -42,6 +42,8 @@ def report_incorrect(file_name, pairs):
 linecount = 0
 pairs = []
 with open(file_name, 'r') as sourcefile:
+    if not re.match(r'#!/usr/bin/env.*', sourcefile.readline(), 0):
+        sourcefile.seek(0)
```

Comment:
How this one is being more general? Would we also accept one line license texts on top of Microsoft copyright notice?